app/assets/javascripts/groups.js

```diff
@@ -0,0 +1,37 @@
+$(document).on('opened', '#collaborators[data-reveal]', function () {
```

Comment:
This is new javascript to autopopulate the groups field when adding a group as a collaborator.